src/components/vm/vmReplaceSpiceDialog.tsx

```diff
@@ -77,9 +77,9 @@ export const ReplaceSpiceDialog = ({
                 // convert them serially, to avoid hammering libvirt with too many parallel requests
                 for (const sel of spiceVMs)
                     if (selected.includes(selectionId(sel)))
-                        await domainReplaceSpice({ connectionName: sel.connectionName, id: sel.id });
+                        await virtXmlEdit(sel, "convert-to-vnc", 1, {});
```

Comment:
Oh, that exists finally! However, still fails on Ubuntu LTS, so maybe it's not there yet?

I'm ok with not offering that feature on 22.04 and 24.04 any more, but it should have some feature test then?